OBTTests/ecl/key/ClassicTestModified.xml

```diff
@@ -0,0 +1,13 @@
+ <Info><Source>user</Source><Message>Training Status: ModelId = 0, Epoch = 1, LR = 1.0, bs = 1024, Loss = 0.00142032571460249</Message></Info>
```

Comment:
Please remove these '<Info>...</Info>' lines from the key file. The Regression Test Engine filters them out.